projects/control-service/projects/pipelines_control_service/src/main/resources/org.apache.tika.mime/custom-mimetypes.xml

```diff
@@ -0,0 +1,5 @@
+<mime-type type="application/pyc">
```

Comment:
That's a good observation. If we have to create a new custom type for each file extension, it might not be suitable. Instead of just adding an additional type to the values.yaml, we would also need to write additional code.

Reply:
Tika doesn't support .pyc files (https://github.com/apache/tika/blob/main/tika-core/src/main/resources/org/apache/tika/mime/tika-mimetypes.xml), which are classified as the most generic binary files - octet-stream, that's why I created this mime type - since this change is focused mainly on filtering the .pyc files produced from Jupiter-notebooks. The test contains a .pyc file.

Reply:
Just as a FYI tika supports more than 1000 file types but not .pyc files.

Reply:
Ok. I am ok if you decide to leave it as it is. 

I feel it might be better to add support for specify extensions. But I am ok if we do not.